src/App.js

```diff
@@ -6,14 +6,27 @@ import createElement from './lib/createElement'
 export default function App() {
   const header = AppHeader('Harry Potter App')
   document.body.append(header)
-
+  const main = createElement('main', { className: 'Card__overview' })
+  document.body.append(main)
   // fetch API
   getCharacters()
     .then(characters => createCards(characters))
     .catch(error => handleGetCharacterError(error))
+
   function createCards(characters) {
-    const cards = characters.map(character => Card(character.name))
-    document.body.append(...cards)
+    const cards = characters.map(character => {
+      const charData = [
+        ['Name', character.name],
+        ['House', character.house],
+        ['Wand length', character.wand.length],
+        ['Ancestry', character.ancestry],
+        ['Date of Birth', character.dateOfBirth],
+      ]
```

Comment:
It's uncommon to use nested Arrays for this kind of data.

```suggestion
      const charData = {
        Name: character.name,
        House: character.house,
        'Wand length': character.wand.length,
        Ancestry: character.ancestry,
        Date of Birth', character.dateOfBirth,
      }
```